### ISSUE TITLE
While typing the name of a steerpoint, window goes unfocused.

### ISSUE THREAD

**Issue by Backspacegun** [aronCiucu/DCSTheWay#123]
While typing the name of a steerpoint, the window sometimes goes unfocused halfway through typing, making me have to click on the name again, which selects the whole text and its hard to unselect everything.

It did it continuously while making a file for my airport waypoints. I now stopped typing names for waypoints for this reason.

**Comment by Backspacegun** [comment]
It seems to unfocus when i stop hovering my mouse over the text input field.
Also everytime i hover over the input field the Icon for the program shows up in taskbar and disappears again when i stop hovering over it.

**Comment by blueberrypi-studio** [comment]
Are you using the latest version (2.1.3)

Personally I've found that The Way works regardless of window focus (since you aren't focused on DCS when you click transfer)

**Comment by aronCiucu** [comment]
Hello, at the moment this is intended, as the window needs focus to be able to type into the field. I decided that hovering over the input field would be a good cue to focus the window

**Comment by blueberrypi-studio** [comment]
Ohhhhh yea that's a good idea, I hadn't encountered that yet

**Comment by Backspacegun** [comment]
> Are you using the latest version (2.1.3)

Yes, i downloaded the program ~30 minutes before i made these issue posts.



> Hello, at the moment this is intended, as the window needs focus to be able to type into the field. I decided that hovering over the input field would be a good cue to focus the window

Aah okay, makes sense.
